_eo_041_enemies/hedetet.md

```diff
@@ -4,7 +4,9 @@ nickname: Hedetet
 image: hedetet.png
 start_floor: 43
 end_floor: 45
-agro: unknown
+agro: Sound
```

Comment:
These are definitely sight. It's in my notes I haven't posted yet and I just double-checked my recording of testing it to be sure